src/components/Header/Header.tsx

```diff
@@ -95,6 +101,7 @@ const Header = () => {
         onClick={() => setExpanded(false)}
         role="presentation"
       />
+      <ThemeToggle />
```

Comment:
nit-pick: we'll want to remove this before it's merged. I know you know just putting a reminder :)